Contributing.md

```diff
@@ -0,0 +1,75 @@
+
+### First steps :feet:
+
+If you like to contribute, the first step is to install Warp from source for devs [steps here](#warp-installation-method-2-from-sourcefor-devs)
```

Comment:
Since files are separated, this link is no longer valid. Please update it to reference the new file